sechub-pds/src/test/java/com/daimler/sechub/pds/test/ExtendedMockMultipartFile.java

```diff
@@ -0,0 +1,38 @@
+package com.daimler.sechub.pds.test;
```

Comment:
SPDX header is missing.

Reply:
I called `apply-headers.sh` and added all missing SPDX headers (there were multiple ones)